eftcamb/06p1_abstract_EFTCAMB_full_map.f90

```diff
@@ -87,7 +87,7 @@ subroutine EFTCAMBFullModelComputeAdotoa( self, a, eft_par_cache, eft_cache )
 
         real(dl)    :: temp
 
-        temp = 1.0_dl/(1.0_dl + eft_cache%EFTOmegaV+ a*eft_cache%EFTOmegaP)*(eft_cache%grhom_t+ 2.0_dl*eft_cache%EFTc -eft_cache%EFTLambda )/3.0_dl
+        temp = 1.0_dl/(1.0_dl + eft_cache%EFTOmegaV+ a*eft_cache%EFTOmegaP)*(eft_cache%grhom_t -eft_cache%grhonu_tot+ 2.0_dl*eft_cache%EFTc -eft_cache%EFTLambda )/3.0_dl
```

Comment:
Here the contribution of massive nu is already included inside the matter components and should not be removed...